src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
```

Comment:
Сreate a variable for base price and use it everywhere 
```suggestion
  const SHORT_TERM_DISCOUNT = 20;
  const BASE_PRICE = days * PRICE_PER_DAY;
  
```

Reply:
ok